README.md

```diff
@@ -9,34 +13,40 @@ npm install
 Run the app:
 
 ```bash
-npx expo start
+npx expo start --dev-client
 ```
 
-You may have to use --tunnel flag:
```

Comment:
any reason you removed this? i still use the --tunnel flag. for me things don't seem to work without it if i have my vpn on.

Reply:
Ok I'll put it back in. I was able to load on my device with using the flag so thought something had changed.

Reply:
actually i'm tripping. expo-dev-client does work without the --tunnel flag for me with my vpn on. prob good to leave it in just in case it's still needed in certain local network situations though.